### PR TITLE
Fix typos and make some edits

### PR DESCRIPTION
**Pull request by joelseq**

### CODE REVIEW

File: _posts/2017-11-15-intro-html-css.md [hunk 1]
```diff
@@ -318,7 +318,7 @@ To style our card divs, let's add these properties into our class selector in CS
 
 ```css
 .card-wrapper {
-  text-align: center; /* Keep our cards centered */
+  text-align: center; /* Keep text in our cards centered */
```
Comment (liawesomesaucer) [inline _posts/2017-11-15-intro-html-css.md]: Wait no this keeps our cards centered too (so our jank % margins don't look off-centered)

File: _posts/2017-11-15-intro-html-css.md [hunk 2]
```diff
@@ -298,7 +298,7 @@ Now, your `gnat.html` page should look something like [this](https://designatucs
 
 ## Cards
 
-Cards are a major feature that are often show up on many sites and help separate different sections of content. In this case, let's try separating the majority of our content, Let's try making our different paragraphs basic cards. Let's assume we've added a couple fables already to our fables website. You can add the next two anchor tags under our original link to `gnat.html`. Let's also wrap our cards in a div.
+Cards are a major feature that often show up on many sites and help separate different sections of content. In this case, let's try separating the different fable links using basic cards. Let's assume we've added a couple fables already to our fables website. You can add the next two anchor tags under our original link to `gnat.html`. Let's also wrap our cards in a div.
```
Comment (liawesomesaucer) [inline _posts/2017-11-15-intro-html-css.md:302]: Good catch!

**Review by liawesomesaucer — CHANGES_REQUESTED** [review]
the comment about centering

File: _posts/2017-11-15-intro-html-css.md (hunk 1 above)
Comment (joelseq) [reply _posts/2017-11-15-intro-html-css.md]: whoops, my bad!

File: _posts/2017-11-15-intro-html-css.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ If you haven't already done so, find and install a text editor. Here are two we
 
 * [Sublime Text](https://www.sublimetext.com/){:target="_blank"}
 * [Atom](https://atom.io/){:target="_blank"}
+* [Visual Studio Code](https://code.visualstudio.com/){:target="_blank"}
```
Comment (joelseq) [inline _posts/2017-11-15-intro-html-css.md:29]: I just had to lol

**Pull request merged**